pipelines/broad/rna_seq/RNAWithUMIsPipeline.wdl

```diff
@@ -0,0 +1,337 @@
+			ribosomal_intervals=ribosomalIntervals,
+			preemptible_tries=0
+	}
+
+	call CollectMultipleMetrics {
+		input:
+			input_bam=UMIAwareDuplicateMarking.duplicate_marked_bam,
+			input_bam_index=UMIAwareDuplicateMarking.duplicate_marked_bam_index,
+			output_bam_prefix=GetSampleName.sample_name,
+			ref_dict=refDict,
+			ref_fasta=ref,
+			ref_fasta_index=refIndex,
+			preemptible_tries=0
+	}
+
+
+  output {
+	File transcriptome_bam = UMIAwareDuplicateMarkingTranscriptome.duplicate_marked_bam
+	File transcriptome_bam_index = UMIAwareDuplicateMarkingTranscriptome.duplicate_marked_bam_index
+	File transcriptome_duplicate_metrics = UMIAwareDuplicateMarkingTranscriptome.duplicate_metrics
+	File output_bam = UMIAwareDuplicateMarking.duplicate_marked_bam
+	File output_bam_index = UMIAwareDuplicateMarking.duplicate_marked_bam_index
+	File duplicate_metrics = UMIAwareDuplicateMarking.duplicate_metrics
+	File gene_tpm = rnaseqc2.gene_tpm
+	File gene_counts = rnaseqc2.gene_counts
+	File exon_counts = rnaseqc2.exon_counts
+	File metrics = rnaseqc2.metrics
+		
+  }
+}
+
+task STAR {
+	input {
+		File bam
+		File starIndex
+	}
+	Int disk_space = ceil(2.2 * size(bam, "GB") + size(starIndex, "GB")) + 250
+
+	command <<<
+		echo $(date +"[%b %d %H:%M:%S] Extracting STAR index")
+		mkdir star_index
+		tar -xvvf ~{starIndex} -C star_index --strip-components=1
+
+		STAR --readFilesIn ~{bam} --readFilesType SAM PE --readFilesCommand samtools view -h \
+			--runMode alignReads --genomeDir star_index --outSAMtype BAM Unsorted --runThreadN 8 \
+			--limitSjdbInsertNsj 1200000 --outSAMstrandField intronMotif --outSAMunmapped Within \
+			--outFilterType BySJout --outFilterMultimapNmax 20 --outFilterScoreMinOverLread 0.33 \
+			--outFilterMatchNminOverLread 0.33 --outFilterMismatchNmax 999 --outFilterMismatchNoverLmax 0.1 \
+			--alignIntronMin 20 --alignIntronMax 1000000 --alignMatesGapMax 1000000 --alignSJoverhangMin 8 \
+			--alignSJDBoverhangMin 1 --alignSoftClipAtReferenceEnds Yes --chimSegmentMin 15 --chimMainSegmentMultNmax 1 \
+			--chimOutType WithinBAM SoftClip --chimOutJunctionFormat 0 --twopassMode Basic --quantMode TranscriptomeSAM --quantTranscriptomeBan Singleend
+
+	>>>
+
+	runtime {
+		docker : "us.gcr.io/tag-team-160914/neovax-tag-rnaseq:v1"
+		disks : "local-disk " + disk_space + " HDD"
+		memory : "64GB"
+		cpu : "8"
+		preemptible: 0
+	}
+
+	output {
+		File aligned_bam = "Aligned.out.bam"
+		File transcriptome_bam = "Aligned.toTranscriptome.out.bam"
+	}
+}
+
+task ExtractUMIs {
+	input {
+		File bam
+		String read1Structure
+		String read2Structure
+	}
+
+	Int disk_space = ceil(2.2 * size(bam, "GB")) + 50
+
+	command <<<
+		fgbio ExtractUmisFromBam --input ~{bam} \
+			--read-structure ~{read1Structure} \
+			--read-structure ~{read2Structure} \
+			--molecular-index-tags RX \
+			--output extractUMIs.out.bam
+	>>>
+
+	runtime {
+		docker : "quay.io/biocontainers/fgbio@sha256:a8e5cf58c318bffba3b2b694a3640ecd9e8106cee2e33b75710c0e8215138b6e"
```

Comment:
Is there an official release for this docker?

Reply:
I'm not sure…

Reply:
https://github.com/fulcrumgenomics/fgbio/releases/tag/1.3.0

Reply:
This docker was updated to one built on the fgbio 1.3.0 release. Ticket here: https://broadinstitute.atlassian.net/browse/GL-1858